Multiple warnings on compile

gcc shows multiple warnings when compiling this module from source codes.
```
$ make
gcc -fPIC -shared -o zabbix_module_docker.so zabbix_module_docker.c -I../../../include -I../../../src/libs/zbxsysinfo
In file included from zabbix_module_docker.c:21:0:
zabbix_module_docker.c: In function ‘zbx_module_docker_socket_query’:
zabbix_module_docker.c:175:43: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘__time_t {aka long int}’ [-Wformat=]
             zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_RCVTIMEO socket timeout: %d seconds", stimeout.tv_sec);
                                           ^                                                    ~~~~~~~~~~
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
zabbix_module_docker.c:179:43: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘__time_t {aka long int}’ [-Wformat=]
             zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_SNDTIMEO socket timeout: %d seconds", stimeout.tv_sec);
                                           ^                                                    ~~~~~~~~~~
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
zabbix_module_docker.c: In function ‘zbx_module_docker_inspect_exec’:
zabbix_module_docker.c:438:62: warning: null argument where non-null required (argument 1) [-Wnonnull]
                                            tofree = string = strdup(get_rparam(request, 3));
                                                              ^~~~~~
zabbix_module_docker.c: In function ‘zbx_module_docker_dev’:
zabbix_module_docker.c:887:37: warning: null argument where non-null required (argument 1) [-Wnonnull]
         char    *stat_file = malloc(strlen(get_rparam(request, 1)) + 2);
                                     ^~~~~~
zabbix_module_docker.c:888:37: warning: null argument where non-null required (argument 1) [-Wnonnull]
         zbx_strlcpy(stat_file, "/", strlen(get_rparam(request, 1)) + 2);
                                     ^~~~~~
zabbix_module_docker.c:889:56: warning: null argument where non-null required (argument 1) [-Wnonnull]
         zbx_strlcat(stat_file, get_rparam(request, 1), strlen(get_rparam(request, 1)) + 2);
                                                        ^~~~~~
In file included from zabbix_module_docker.c:21:0:
zabbix_module_docker.c:949:45: warning: format ‘%d’ expects argument of type ‘int’, but argument 6 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
                 zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; stat file: %s, metric: %s; value: %d", container, stat_file, metric, value);
                                             ^
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
zabbix_module_docker.c: In function ‘zbx_module_docker_mem’:
zabbix_module_docker.c:1056:45: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
                 zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %d", container, metric, value);
                                             ^
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
zabbix_module_docker.c: In function ‘zbx_module_docker_cpu’:
zabbix_module_docker.c:1201:45: warning: format ‘%d’ expects argument of type ‘int’, but argument 5 has type ‘uint64_t {aka long unsigned int}’ [-Wformat=]
                 zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %d", container, metric, result_value);
                                             ^
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
zabbix_module_docker.c: In function ‘zbx_module_docker_net’:
zabbix_module_docker.c:1356:43: warning: format ‘%s’ expects a matching ‘char *’ argument [-Wformat=]
             zabbix_log(LOG_LEVEL_WARNING, "Cannot execute netns command: %s");
                                           ^
../../../include/log.h:67:28: note: in definition of macro ‘zabbix_log’
    __zbx_zabbix_log(level, __VA_ARGS__);      \
                            ^~~~~~~~~~~
```

Not a bug but need to fix this.

--- a/src/modules/zabbix_module_docker/zabbix_module_docker.c
+++ b/src/modules/zabbix_module_docker/zabbix_module_docker.c
@@ -435,8 +435,8 @@ struct inspect_result     zbx_module_docker_inspect_exec(AGENT_REQUEST *request)
                                            zbx_strlcpy(value, p_array+1, s_size);
                                            zabbix_log(LOG_LEVEL_DEBUG, "Array item: %s", value);
 
-                                           char	*arg3 = get_rparam(request, 3);
-                                           tofree = string = strdup(arg3);
+                                           char	  *arg4 = get_rparam(request, 3);
+                                           tofree = string = strdup(arg4);
 
                                            // hacking: Marathon MESOS_TASK_ID, Chronos - mesos_task_id
                                            // docker.inspect[cid,Config,Env,MESOS_TASK_ID=|mesos_task_id=]
@@ -884,9 +884,9 @@ int     zbx_module_docker_dev(AGENT_REQUEST *request, AGENT_RESULT *result)
         }
 
         container = zbx_module_docker_get_fci(get_rparam(request, 0));
-        char    *arg1 = get_rparam(request, 1);
-        char    *stat_file = malloc(strlen(arg1) + 2);
-        zbx_strlcpy(stat_file, "/", strlen(arg1) + 2);
+        char    *arg2 = get_rparam(request, 1);
+        char    *stat_file = malloc(strlen(arg2) + 2);
+        zbx_strlcpy(stat_file, "/", strlen(arg2) + 2);
         zbx_strlcat(stat_file, get_rparam(request, 1), strlen(arg1) + 2);
         metric = get_rparam(request, 2);
 

--- a/src/modules/zabbix_module_docker/zabbix_module_docker.c
+++ b/src/modules/zabbix_module_docker/zabbix_module_docker.c
@@ -887,7 +887,7 @@ int     zbx_module_docker_dev(AGENT_REQUEST *request, AGENT_RESULT *result)
         char    *arg2 = get_rparam(request, 1);
         char    *stat_file = malloc(strlen(arg2) + 2);
         zbx_strlcpy(stat_file, "/", strlen(arg2) + 2);
-        zbx_strlcat(stat_file, get_rparam(request, 1), strlen(arg1) + 2);
+        zbx_strlcat(stat_file, get_rparam(request, 1), strlen(arg2) + 2);
         metric = get_rparam(request, 2);
 
         char    *cgroup = "blkio/";

--- a/src/modules/zabbix_module_docker/zabbix_module_docker.c
+++ b/src/modules/zabbix_module_docker/zabbix_module_docker.c
@@ -435,7 +435,8 @@ struct inspect_result     zbx_module_docker_inspect_exec(AGENT_REQUEST *request)
                                            zbx_strlcpy(value, p_array+1, s_size);
                                            zabbix_log(LOG_LEVEL_DEBUG, "Array item: %s", value);
 
-                                           tofree = string = strdup(get_rparam(request, 3));
+                                           char	*arg3 = get_rparam(request, 3);
+                                           tofree = string = strdup(arg3);
 
                                            // hacking: Marathon MESOS_TASK_ID, Chronos - mesos_task_id
                                            // docker.inspect[cid,Config,Env,MESOS_TASK_ID=|mesos_task_id=]
@@ -883,9 +884,10 @@ int     zbx_module_docker_dev(AGENT_REQUEST *request, AGENT_RESULT *result)
         }
 
         container = zbx_module_docker_get_fci(get_rparam(request, 0));
-        char    *stat_file = malloc(strlen(get_rparam(request, 1)) + 2);
-        zbx_strlcpy(stat_file, "/", strlen(get_rparam(request, 1)) + 2);
-        zbx_strlcat(stat_file, get_rparam(request, 1), strlen(get_rparam(request, 1)) + 2);
+        char    *arg1 = get_rparam(request, 1);
+        char    *stat_file = malloc(strlen(arg1) + 2);
+        zbx_strlcpy(stat_file, "/", strlen(arg1) + 2);
+        zbx_strlcat(stat_file, get_rparam(request, 1), strlen(arg1) + 2);
         metric = get_rparam(request, 2);
 
         char    *cgroup = "blkio/";

--- a/src/modules/zabbix_module_docker/zabbix_module_docker.c
+++ b/src/modules/zabbix_module_docker/zabbix_module_docker.c
@@ -172,11 +172,11 @@ const char*  zbx_module_docker_socket_query(char *query, int stream)
         // socket input/output timeout
         if (setsockopt (sock, SOL_SOCKET, SO_RCVTIMEO, (char *)&stimeout, sizeof(stimeout)) < 0)
         {
-            zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_RCVTIMEO socket timeout: %d seconds", stimeout.tv_sec);
+            zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_RCVTIMEO socket timeout: %ld seconds", stimeout.tv_sec);
         }
         if (setsockopt (sock, SOL_SOCKET, SO_SNDTIMEO, (char *)&stimeout, sizeof(stimeout)) < 0)
         {
-            zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_SNDTIMEO socket timeout: %d seconds", stimeout.tv_sec);
+            zabbix_log(LOG_LEVEL_WARNING, "Cannot set SO_SNDTIMEO socket timeout: %ld seconds", stimeout.tv_sec);
         }
 
         temp1 = string_replace(query, "\n", "");
@@ -945,7 +945,7 @@ int     zbx_module_docker_dev(AGENT_REQUEST *request, AGENT_RESULT *result)
                          break;
                      }
                 }
-                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; stat file: %s, metric: %s; value: %d", container, stat_file, metric, value);
+                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; stat file: %s, metric: %s; value: %lu", container, stat_file, metric, value);
                 SET_UI64_RESULT(result, value);
                 ret = SYSINFO_RET_OK;
                 break;
@@ -1052,7 +1052,7 @@ int     zbx_module_docker_mem(AGENT_REQUEST *request, AGENT_RESULT *result)
                         zabbix_log(LOG_LEVEL_ERR, "sscanf failed for matched metric line");
                         continue;
                 }
-                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %d", container, metric, value);
+                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %lu", container, metric, value);
                 SET_UI64_RESULT(result, value);
                 ret = SYSINFO_RET_OK;
                 break;
@@ -1197,7 +1197,7 @@ int     zbx_module_docker_cpu(AGENT_REQUEST *request, AGENT_RESULT *result)
                         result_value /= cpu_num;
                 }
 
-                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %d", container, metric, result_value);
+                zabbix_log(LOG_LEVEL_DEBUG, "Id: %s; metric: %s; value: %lu", container, metric, result_value);
                 SET_UI64_RESULT(result, result_value);
         }
         free(container);
@@ -1352,7 +1352,7 @@ int     zbx_module_docker_net(AGENT_REQUEST *request, AGENT_RESULT *result)
         free(netns);
         if (fp == NULL)
         {
-            zabbix_log(LOG_LEVEL_WARNING, "Cannot execute netns command: %s");
+            zabbix_log(LOG_LEVEL_WARNING, "Cannot execute netns command");
             SET_MSG_RESULT(result, zbx_strdup(NULL, "Cannot execute netns command"));
             return SYSINFO_RET_FAIL;
         }

